background/addon-state.js

```diff
@@ -84,7 +106,7 @@ class AddonState {
       this.state.tabColorMap.delete(tabId));
 
     this.refreshAddon = async () => {
-      onInit();
+      onInit(this.state);
```

Comment:
You don't need to pass `this.state`, you can just refer it from `onInit` as `this.state` (the way this works is the `onInit = onInit.bind(this)` above in this file.

Reply:
Oh okay, I'm really not experienced with JavaScript so I didn't know that. Changed it.